deploy/basic/README.md

```diff
@@ -1,8 +1,12 @@
 ![MuShop Logo](../../images/logo.png)
 
-This is a Terraform configuration that deploys the MuShop Basic demo on [Oracle Cloud Infrastructure (OCI)][oci].
+This is a Terraform configuration that deploys the MuShop application demo on [Oracle Cloud Infrastructure (OCI)][oci].
 
-MuShop Basic is a sample 3-tier web application that implements an e-commerce site. It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci]. This sample is designed to run using only the Always Free tier resources. This sample contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack. This stack creates all the required resources and configures the application on the created resources.
+MuShop is a 3-tier web application that implements an e-commerce site. 
+It is built to showcase the features of [Oracle Cloud Infrastructure (OCI)][oci].
+This application is designed to run using only the Always Free tier resources. 
```

Comment:
Application is designed to run using only the Always Free tier resources. The repository contains the application code as well as the [Terraform][tf] code to create a [Resource Manager][orm] stack, that creates all the required resources and configures the application on the created resources.

Reply:
done